skunk.gemspec

```diff
@@ -41,7 +41,6 @@ Gem::Specification.new do |spec|
   spec.add_dependency "rubycritic", ">= 4.4", "< 5.0"
   spec.add_dependency "terminal-table", "~> 1.8.0"
 
-  spec.add_development_dependency "bundler", "~> 2.0"
```

Comment:
Why we dropping the bundler dependency?

Reply:
The main reason was because Travis wasn't building, check it out here:

https://travis-ci.org/github/fastruby/skunk/builds/728404194

Then there's the question about if `bundler` _is_ a development dependency of Skunk which I intuitively thought it isn't.  

Honestly I'm not sure what the usual pattern and if there's a reason why one would specify it as a dep. Any ideas?